uploaded/file.go

```diff
@@ -4,14 +4,19 @@ import (
 	"mime/multipart"
 )
 
-// File represents a file and it's name & content-type
-type File struct {
+// FileMetadata represents the metadata that comes with an uploaded file
+type FileMetadata struct {
 	Filename    string
 	ContentType string
-	Data        multipart.File
 }
 
-// NewFile creates a new uploade file instance
-func NewFile(filename string, contentType string, file multipart.File) *File {
-	return &File{Filename: filename, ContentType: contentType, Data: file}
+// File represents a file and it's name & content-type
+type File struct {
```

Comment:
can this be derived from the `File` MAP in some way, instead of recreating here?

Reply:
No, this is just the subset of that information that gets transferred into s3 (or some other storage that we may choose in the future)